widget/widget.go

```diff
@@ -80,6 +81,28 @@ func (w *baseWidget) hide(parent fyne.Widget) {
 	canvas.Refresh(parent)
 }
 
+func (w *baseWidget) Enabled() bool {
```

Comment:
Can we do without this? (or can it be unexported?)
It implies (just in reading, I know it is not technically true) that all baseWidgets are disableable.
It may be minor here but if we expose as BaseWidget this would be undesirable.

Reply:
Yeah, that makes sense.  I think I was just going for an analog to the Visible public method but it isn't quite the same.  Fixed, pushed.